tofd/calibration/R3BTofDCal2Hit.cxx

```diff
@@ -191,13 +192,13 @@ InitStatus R3BTofDCal2Hit::Init()
         return kFATAL;
     }
 
-    header = dynamic_cast<R3BEventHeader*>(mgr->GetObject("EventHeader."));
+    header = (R3BEventHeader*)mgr->GetObject("EventHeader.");
     R3BLOG_IF(fatal, NULL == header, "EventHeader. not found");
 
-    fCalItems = dynamic_cast<TClonesArray*>(mgr->GetObject("TofdCal"));
+    fCalItems = (TClonesArray*)mgr->GetObject("TofdCal");
     R3BLOG_IF(fatal, NULL == fCalItems, "TofdCal not found");
 
-    fCalTriggerItems = dynamic_cast<TClonesArray*>(mgr->GetObject("TofdTriggerCal"));
+    fCalTriggerItems = (TClonesArray*)mgr->GetObject("TofdTriggerCal");
     R3BLOG_IF(fatal, NULL == fCalTriggerItems, "TofdTriggerCal not found");
```

Comment:
I don't quite understand here. Why do you change dynamic_cast back to C-style cast?

Reply:
This is something general in the PR, I guess Enis started with these improvements some months ago before the current implementation based on "dynamic_cast".

Reply:
I see. But should the current PR be rebased on the HEAD of dev branch?

Reply:
Yes, rebase the dev branch is the best